sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/InsertAdaptiveSparkPlan.scala

```diff
@@ -44,7 +45,7 @@ case class InsertAdaptiveSparkPlan(
 
   private def applyInternal(plan: SparkPlan, isSubquery: Boolean): SparkPlan = plan match {
     case _ if !conf.adaptiveExecutionEnabled => plan
-    case _: ExecutedCommandExec => plan
+    case _: ExecutedCommandExec | _: V2CommandExec => plan
```

Comment:
ideally this is corrected, but then we need to apply AQE for input queries of command. e.g. CTAS, INSERT